theme/icons/three-vertical-dots.svg

```diff
@@ -0,0 +1 @@
+<svg width="10" height="20" xmlns="http://www.w3.org/2000/svg"><g><circle cx="9.5" cy="4.5" r="1.5"/><circle cx="9.5" cy="10.5" r="1.5"/><circle cx="9.5" cy="16.5" r="1.5"/></g></svg>
```

Comment:
We are not using `width`/`height` property for icons, let's change it to `viewBox` (with 0 0 20 20).

 Also, I would take into consideration to transform svg content to `<path>` instead a group of `<circle>`:

Reply:
Is path any better than circle @dkonopka?

Reply:
I was thinking about the smaller weight of `SVG` file, but I'm not so sure.